### ISSUE TITLE
setting for  helm-split-window-inside-p not respected?

### ISSUE THREAD

**Issue by rdiaz02** [emacs-helm/helm-mu#52]
I have `(setq helm-split-window-inside-p t)` but this does not seem to be honored with `helm-mu` : as soon as I run `helm-mu` it uses the whole frame (so even windows that are unrelated to mu4e are not shown while the search is ongoing). Is there a way to  prevent helm-mu from using the complete frame (i.e., to have helm-mu use only a split in the header window)?

**Comment by tmalsburg** [comment]
Full-frame is currently hard-coded (see [here](https://github.com/emacs-helm/helm-mu/blob/master/helm-mu.el#L524)).  Happy to change it, but do you know how users can configure full-frame just for helm-mu?

**Comment by rdiaz02** [comment]
> Full-frame is currently hard-coded (see [here](https://github.com/emacs-helm/helm-mu/blob/master/helm-mu.el#L524)). 

Ooops, I wonder how I searched the code. Sorry for missing such obvious thing.

> Happy to change it, but do you know how users can configure full-frame just for helm-mu?

Thanks! Unfortunately, I don't. But while searching for it, I realized I can get the behavior I want by simply setting:
```
(setq helm-display-function #'pop-to-buffer)
```
which splits the current `*mu4e-main*` or the `*mu4e-headers*` window, leaving the rest of the windows unaffected.

More sophisticated behavior is possible with [shackle](https://depp.brause.cc/shackle/) which is what I am using now as:
```
(setq helm-display-function #'pop-to-buffer)
(setq shackle-rules '(("*helm mu*" :regexp nil
		       :same t
		       )))
(shackle-mode)
```
This uses the whole `*mu4e-main*` or the whole `*mu4e-headers*` window for `helm-mu`. 

Since it seems this can be easily solved, I am closing the issue. Apologies for the noise.
